p2p/pre2p/handlers.go

```diff
@@ -0,0 +1,38 @@
+package pre2p
+
+import (
+	"io/ioutil"
+	"log"
+	"net"
+
+	"github.com/pokt-network/pocket/shared/types"
+
+	"google.golang.org/protobuf/proto"
+)
+
+func (m *p2pModule) handleNetworkMessage(conn net.Conn) {
+	defer conn.Close()
+
+	data, err := ioutil.ReadAll(conn)
+	if err != nil {
+		log.Println("Error reading from conn: ", err)
+		return
+	}
+
+	networkMessage := types.PocketEvent{}
+	if err := proto.Unmarshal(data, &networkMessage); err != nil {
+		panic(err)
+	}
+	if err != nil {
+		log.Println("Error decoding network message: ", err)
+		return
+	}
+
+	event := types.PocketEvent{
+		Module: types.SourceModule_P2P_MODULE,
```

Comment:
I wonder if a `Module` property is necessary on an event. A topic should imply enough context. However, I see how useful can this be for debugging purposes.

Noting for later, this can benefit from some thought.

Reply:
Removing. We can always add it back later if need be.